### DISCUSSION TITLE
Upgrades UniFi Network Application to 8.4.59

### DISCUSSION THREAD

**Discussion by rami-v** [hassio-addons/addon-unifi#521] [General]
Hi all
unifi release version 8.4.59 - can you please update the add on in order to let us option to upgrade in the home assistant system

is this on the roadmap ?

**Comment by dobrac** [comment]
Hi, I opened a PR last week (https://github.com/hassio-addons/addon-unifi/pull/519), which now waits (based on previous updates) for Frenck to approve and merge.

**Comment by rami-v** [comment]
Hi
unifi released version 8.4.62
this update will be available soon for the HAOS users?

thanks....

**Reply by cfergeau** [reply]
8.4.62 update was proposed in https://github.com/hassio-addons/addon-unifi/pull/522
Waiting for review/approval/merge before being available for users.

**Comment by frenck** [comment]
There are PRs for these kind of things. Closing this discussion.
